src/Common/tests/XunitTraitsDiscoverers/PlatformID.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit.Sdk;
+
+namespace Xunit
+{
+    [Flags]
+    public enum PlatformID
+    {
+        Windows = 1,
+        Linux = 2,
+        OSX = 4,
+        Any = 8
```

Comment:
I included Any, in case more platforms are added, and in code PlatformID.Any is easier than doing, PlatformID.Windows | PlatformID.Linux | PlatformID.OSX..